static/js/compose_actions.js

```diff
@@ -136,6 +136,30 @@ export function expand_compose_box() {
     $(".message_comp").show();
 }
 
+function composing_to_current_topic_narrow() {
+    const stream_name = compose_state.stream_name();
+    const topic = compose_state.topic() || undefined;
+    return narrow_state.stream() === stream_name && narrow_state.topic() === topic;
+}
```

Comment:
This isn't a blocker for test-deploying this, but we should use `util.same_topic` or similar to do this check; topics and stream names are both case-insensitive.

Reply:
Ok, changed to
```js
function composing_to_current_topic_narrow() {
    return (
        util.lower_same(compose_state.stream_name(), narrow_state.stream() || "") &&
        util.lower_same(compose_state.topic(), narrow_state.topic())
    );
}